pkgs/applications/terminal-emulators/rio/default.nix

```diff
@@ -26,6 +26,9 @@
 
 , withWayland ? !stdenv.isDarwin
 , wayland
+
+, testers
+, rio
 }:
 let
   rlinkLibs = if stdenv.isDarwin then [
```

Comment:
We probably just need to add MetalKit here

Reply:
That was it indeed.
However we have a test failing:

```
failures:

---- corcovado/src/sys/unix/eventedfd.rs - sys::unix::eventedfd::EventedFd (line 31) stdout ----
Test executable failed (exit status: 101).

stderr:
thread 'main' panicked at corcovado/src/sys/unix/eventedfd.rs:24:16:
called `Result::unwrap()` on an `Err` value: Os { code: 1, kind: PermissionDenied, message: "Operation not permitted" }
stack backtrace:
   0:        0x104d4b838 - <std::sys_common::backtrace::_print::DisplayBacktrace as core::fmt::Display>::fmt::h461bbd67a6621422
   1:        0x104ce5828 - core::fmt::write::heed46d935b88c574
   2:        0x104d4e6a0 - std::io::Write::write_fmt::h059fcaa37c7fb6b7
   3:        0x104d4b5d8 - std::sys_common::backtrace::print::he45faad0cc62205b
   4:        0x104d4c700 - std::panicking::default_hook::{{closure}}::h079849c23e14ba8c
   5:        0x104d4c268 - std::panicking::default_hook::h23406b9c4937065d
   6:        0x104d4d330 - std::panicking::rust_panic_with_hook::h590310b8f01355ea
   7:        0x104d4ce8c - std::panicking::begin_panic_handler::{{closure}}::h1e72464f58135f03
   8:        0x104d4bb38 - std::sys_common::backtrace::__rust_end_short_backtrace::h569fec281ca4c9f5
   9:        0x104d4cae8 - _rust_begin_unwind
  10:        0x104d52534 - core::panicking::panic_fmt::h78e83b1750882f24
  11:        0x104d529c4 - core::result::unwrap_failed::h4109b0718192d6d7
  12:        0x104cf5710 - rust_out::main::h4fb46e86f974d2f5
  13:        0x104cf3c34 - core::ops::function::FnOnce::call_once::hf02b6f540bebbb60
  14:        0x104cf2fec - std::sys_common::backtrace::__rust_begin_short_backtrace::ha5dea535a3027ddc
  15:        0x104cf3690 - std::rt::lang_start::{{closure}}::hcd09604f56e8968b
  16:        0x104d4c7c0 - std::panicking::try::h306a92dbe5fcd94d
  17:        0x104d42bd4 - std::rt::lang_start_internal::hcd3b15c34f03da8c
  18:        0x104cf3668 - std::rt::lang_start::h809b4565db301f43
  19:        0x104cf576c - _main



failures:
    corcovado/src/sys/unix/eventedfd.rs - sys::unix::eventedfd::EventedFd (line 31)

test result: FAILED. 55 passed; 1 failed; 0 ignored; 0 measured; 0 filtered out; finished in 12.66s
```

Reply:
Maybe it tries to access the GUI ?

Reply:
I don't know, just leave it marked broken